src/SFA.DAS.AANHub.Database/Scripts/AddMemberData.sql

```diff
@@ -0,0 +1,25 @@
+﻿CREATE TABLE #TempMember
+(
+	[Id] uniqueidentifier,
+    [UserType] NVARCHAR(10),
+	[Status] NVARCHAR(10),
+	[RegionId] INT,
```

Comment:
RegionId can be removed now, I think.  As well as from the Members table. We have a navigation property on `Member` entity that adds to MembersRegion if provided.

Reply:
All reference to RegionId has gone from the new seed script. Just want to make sure you want RegionId column removed in  Member.sql too